src/controllers/restful/PaymentController.ts

```diff
@@ -345,7 +345,7 @@ export function getResponseErrorIfExists(
     faultBean.faultCode
   );
   return ResponseErrorGeneric(
```

Comment:
you should user ResponseErrorInternal now

Reply:
With ResponseErrorInternal it can just send a text message.

Here it should send more information to backend:
```
return ResponseErrorGeneric(
    HttpStatusCodeEnum.HTTP_STATUS_500,
    errorMessageCtrl,
    faultBean.faultCode,
    faultBean.faultString
  );
```

Otherwise, actually I'm setting errorMessageCtrl as title. If I use ResponseErrorInternal, just a detail could be specified. So, proxy code should be reviewed again.